File: 백준/Bronze/4101. 크냐？/크냐？.js
let input = require('fs').readFileSync('/dev/stdin').toString().trim().split('\n');
input.pop();
let result = input.reduce((acc, cur) => {
    let [a, b] = cur.split(' ').map(el => parseInt(el));
    if(a > b) acc.push("Yes");
    else acc.push("No");
    return acc
}, []);

console.log(result.join('\n'));

File: 백준/Bronze/4101. 크냐？/크냐？.c
#include <stdio.h>

int main() {
    while (1) {
        int a, b;
        scanf("%d %d", &a, &b);

        if (a == 0 && b == 0) break;
        if (a > b) printf("Yes\n");
        else printf("No\n");
    }
    return 0;
}